python/paddle/fluid/executor.py

```diff
@@ -1203,7 +1203,7 @@ def _run_impl(self, program, feed, fetch_list, feed_var_name,
                 if vardesc.persistable() == False and \
                     vardesc.type() == core.VarDesc.VarType.LOD_TENSOR and \
                     vardesc.need_check_feed() == True and \
-                    varobj._stop_gradient == True and \
+                    varobj.stop_gradient == True and \
```

Comment:
This is old code, so it is not your fault and you have nothing to change. However, I would like you to know it, there is a coding style recommendation in Python: don't compare boolean values to `True` or `False` using `==`:

![7e22ef7f0594d5938d0160d254b790ab](https://user-images.githubusercontent.com/7913861/126252192-15784b5d-603b-461b-ac7d-a3c398640e2c.png)


Reference:
https://www.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
Thanks for reminder!